src/python/WMCore/ReqMgr/Service/Auxiliary.py

```diff
@@ -169,7 +171,10 @@ def get(self, subName=None):
         """
         try:
             if subName:
-                docName = "%s_%s" % (self.name, subName)
+                if subName == "ALL_DOCS":
```

Comment:
To be on a safe side, may be here we should use
```
if subName.lower() == 'all_docs':
```

Reply:
Is it a concern that something might change the arguments encoded in the url? Or just to make it easier to people interacting with this API?